Lib/ufo2ft/postProcessor.py

```diff
@@ -357,6 +361,99 @@ def _subroutinize_with_cffsubr(cls, otf, cffVersion):
 
         return cffsubr.subroutinize(otf, cff_version=cffVersion, keep_glyph_names=False)
 
+    def apply_fontinfo(self):
+        """Apply the fontinfo data from the DesignSpace variable-font's lib to
+        the compiled font."""
+        import copy
+
+        # Create a temporary UFO and sets its fontinfo to the union of the main
+        # UFO's fontinfo and the DesignSpace variable-font’s info.
+        temp_ufo = type(self.ufo)()
+        temp_ufo.info = copy.copy(self.ufo.info)
+        for k, v in self.info.items():
+            setattr(temp_ufo.info, k, v)
+
+        # Build a temporary font with the only tables that can be modified with
+        # fontinfo.
+        tables = {"head", "hhea", "name", "OS/2", "post"} & set(self.otf.keys())
```

Comment:
aren't you missing vhea? and what about the fontinfo attributes that start with `postscript` and which contribute to define CFF stuff?

Reply:
and gasp as well

Reply:
> and what about the fontinfo attributes that start with `postscript` and which contribute to define CFF stuff?

Is there anything that is relevant to variable fonts, aren’t these dropped in `CFF2` table?

Reply:
> Is there anything that is relevant to variable fonts, aren’t these dropped in CFF2 table?

I don't know much about those to be honest. Let's address this if/when someone request